operators/operator-reference.adoc

```diff
@@ -6,6 +6,14 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 include::modules/cloud-credential-operator.adoc[leveloffset=+1]
```

Comment:
These changes have been reviewed in the other PR for 4.7 CCO work, but I copied them over since that one isn't merged yet.